terraform/modules/networking/ecr_interfaces.tf

```diff
@@ -53,11 +53,17 @@ data "aws_iam_policy_document" "ecr_dkr_endpoint" {
       "ecr:BatchCheckLayerAvailability",
       "ecr:GetDownloadUrlForLayer",
       "ecr:BatchGetImage",
+      # For ECR Public pull through cache
+      "ecr:BatchImportUpstreamImage",
+      "ecr:CreateRepository",
```

Comment:
To make it clearer that the comment applies to both lines, I'd put it on the right of each of them.